.codeclimate.yml

```diff
@@ -22,9 +22,4 @@ checks:
   identical-code:
     enabled: false
 
-plugins:
-  rubocop:
-    enabled: true
-    channel: rubocop-0-74
-
 exclude_patterns: []
```

Comment:
Doesn't really look like we use Code Climate at all since removing rubocop, should we just delete this file? cc @rafaelfranca

Reply:
I agree to disable Code Climate but did not remove .codeclimate.yml file in this pull request because as far as I know removing .codeclimate.yml does not disable Code Climate, it would restore the default maintainability checks disabled by #31292.

Refer rsim/oracle-enhanced#1938, removing .codeclimate.yml causes "codeclimate — 83 issues to fix".

Reply:
Yeah. We need to disable code cliemate in the web site. Let me see if I can do.

Reply:
I deleted to hook. We can remove this file now.

Reply:
Thanks for taking care of disabling Code Climate at the web console. I have removed `.codeclimate.yml` file.